OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -15,6 +15,13 @@
 #pragma clang diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
 #endif
 
+#define PRINT(msg) PLATFORM_PRINT(msg)
+
+#define PRINTF(format, ...)                                \
```

Comment:
I fear that this is going to be abused. The way I see it every print should be associated with file and line (and possibly timestamp in the future) and hence every print should use LOG.

Reply:
Yeah that makes sense.  I'll convert to LOG.